apps/meteor/ee/server/api/sessions.ts

```diff
@@ -0,0 +1,173 @@
+import { Users, Sessions } from '@rocket.chat/models';
+import { IUser } from '@rocket.chat/core-typings';
+import { escapeRegExp } from '@rocket.chat/string-helpers';
+
+import { isSessionsPaginateProps, isSessionsProps } from '../../definition/rest/v1/sessions';
+import { API } from '../../../app/api/server/api';
+import { hasLicense } from '../../app/license/server/license';
+import { Notifications } from '../../../app/notifications/server';
+
+const validateSortKeys = (sortKeys: string[]): boolean => {
+	const validSortKeys = ['loginAt', 'device.name', 'device.os.name', 'device.os.version', '_user.name', '_user.username'];
+	const invalidSortKeys = sortKeys.filter((key) => !validSortKeys.includes(key));
+
+	if (invalidSortKeys.length) {
+		return false;
+	}
```

Comment:
A suggestion is that you can use .some for scenarios like this, and that won't create a new array, would just return true/false